testlib/src/main/java/io/spine/testing/Testing.java

```diff
@@ -82,4 +97,125 @@ public static void repeat(int count, Runnable action) {
     public static void halt() throws AssertionError {
         fail("This method should never be called.");
     }
+
+    /**
+     * Throws {@code IllegalStateException} with the formatted string and the cause.
+     *
+     * @param cause the cause of the exception
+     * @param format the format string
+     * @param args formatting parameters
+     * @return nothing ever, always throws an exception. The return type is given for convenience.
+     * @throws IllegalStateException always
+     */
+    @CanIgnoreReturnValue
+    public static IllegalStateException newIllegalStateException(Throwable cause,
```

Comment:
I'd like to propose two things.

1. Instead of copying the "throwing" code all the way, we could just throw corresponding exceptions by hand.

We don't have many cases in `testlib`, so I would say it's OK.

The main reason here is that eventually, users will have two JARs in their classpath: `testlib` and `base`. In this case, we'll have some ambiguity on imports.

2. For the rest of methods — for those which don't exist just to throw — mark the `Testing` type `@Internal`.